package.json

```diff
@@ -1,14 +1,19 @@
 {
   "license": "Apache-2.0",
   "private": true,
-  "workspaces": [
-    "packages/client",
-    "packages/common",
-    "packages/main"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/client",
+      "packages/common",
+      "packages/main"
+    ],
+    "nohoist": [
+      "**/*"
```

Comment:
When using yarn workspaces normally dependencies are installed in the `node_modules/` directory in the project root, and all of the packages in the repository look for dependencies there. Unfortunately Electron Forge (which is run from the main package) is not set up to search parent directories for `node_modules/`. The `nohoist` configuration here instructs yarn to put all dependencies in a `node_modules/` directory in the respective package directory.